### ISSUE TITLE
Version 2.0.15 introduces a new bug on Debian 9.x / Rasbian

### ISSUE THREAD

**Issue by ghost** [DNSCrypt/dnscrypt-proxy#478]
```
[2018-06-06 18:31:39] [NOTICE] Source [public-resolvers.md] loaded
[2018-06-06 18:31:39] [NOTICE] dnscrypt-proxy 2.0.15
[2018-06-06 18:31:39] [NOTICE] Now listening to 10.0.0.24:53 [UDP]
[2018-06-06 18:31:39] [NOTICE] Now listening to 10.0.0.24:53 [TCP]
[2018-06-06 18:31:39] [NOTICE] [dnscrypt.nl-ns0] OK (crypto v1) - rtt: 16ms
[2018-06-06 18:31:39] [NOTICE] Server with the lowest initial latency: dnscrypt.nl-ns0 (rtt: 16ms)
[2018-06-06 18:31:39] [NOTICE] dnscrypt-proxy is ready - live servers: 1
panic: interface conversion: interface {} is *net.TCPConn, not *net.Conn

goroutine 13 [running]:
main.(*Proxy).exchangeWithTCPServer(0x12526014, 0x125ea000, 0x125ea041, 0x126e8200, 0x100, 0x100, 0x127de090, 0xc, 0xc, 0x100, ...)
        /home/travis/gopath/src/github.com/jedisct1/dnscrypt-proxy/dnscrypt-proxy/proxy.go:224 +0x414
main.(*Proxy).processIncomingQuery(0x12526014, 0x125ea000, 0x43128f, 0x3, 0x43128f, 0x3, 0x1240f302, 0x1f, 0x1000, 0x127c4048, ...)
        /home/travis/gopath/src/github.com/jedisct1/dnscrypt-proxy/dnscrypt-proxy/proxy.go:302 +0x998
main.(*Proxy).tcpListener.func1(0x127c4008, 0x12526014)
        /home/travis/gopath/src/github.com/jedisct1/dnscrypt-proxy/dnscrypt-proxy/proxy.go:184 +0x1f0
created by main.(*Proxy).tcpListener
        /home/travis/gopath/src/github.com/jedisct1/dnscrypt-proxy/dnscrypt-proxy/proxy.go:171 +0xa8
```

Edit:

```
-- System Information:
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 9.4 (stretch)
Release:        9.4
Codename:       stretch
Architecture: armv7l

Kernel: Linux 4.14.34-v7+ (SMP w/4 CPU cores)
Locale: LANG=en_GB.UTF-8, LC_CTYPE=en_GB.UTF-8 (charmap=UTF-8), LANGUAGE=en_GB.UTF-8 (charmap=UTF-8)
Shell: /bin/sh linked to /bin/dash
Init: systemd (via /run/systemd/system)

```

Everything fine on the previous version, I was forced to roll back right away. If you need me to test something to find the error together I will help!

**Comment by jedisct1** [comment]
Thanks!

There are still basic things I don't understand about Go.

This should be fixed now, and new binaries will appear soon.

Thanks!

**Comment by ilovezfs** [comment]
@jedisct1 in the future, please bump the version number rather than replacing existing release artifacts. In Homebrew, we'd already shipped 2.0.15 so users are now getting checksum errors due to the altered sources.
